Realm/Tests/ObjectTests.m

```diff
@@ -785,7 +785,7 @@ - (void)testCreateInRealmValidationForDictionary {
         }
 
         if ([keyToInvalidate isEqualToString:@"anyCol"]) {
-            obj = self;
+            continue;
         }
```

Comment:
remove this

Reply:
we want to skip checking invalid value for anyCol, given that it accepts List and Dictionary